Sources/SwiftInspectorCommands/ImportsCommand.swift

```diff
@@ -41,13 +42,14 @@ final class ImportsCommand: ParsableCommand {
   /// Runs the command
   func run() throws {
     let cachedSyntaxTree = CachedSyntaxTree()
-    let analyzer = ImportsAnalyzer(cachedSyntaxTree: cachedSyntaxTree)
+    let analyzer = StandardAnalyzer(cachedSyntaxTree: cachedSyntaxTree)
     let fileURL = URL(fileURLWithPath: path)
 
     let outputArray = try FileManager.default.swiftFiles(at: fileURL)
       .reduce(Set<String>()) { result, url in
-        let importStatements = try analyzer.analyze(fileURL: url)
-        let output = importStatements.map { outputString(from: $0) }
+        let reader = ImportSyntaxReader()
+        try analyzer.analyze(fileURL: url, withVisitor: reader)
```

Comment:
I'm not sure about this change. This is making `ImportSyntaxReader` which is a subclass of `SyntaxRewriter` visible to the Command, making the boundary a lot more blurry

How do you feel about having a factory that creates these analyzers internally in the Analyzer layer and then we keep the `analyze` method just taking a fileURL?

That way we can make the StandardAnalyzer usage of SwiftSyntax information internal.